### PR TITLE
wg-netmanager: init at 0.2.8

### PR DESCRIPTION
**Pull request by gin66**
###### Motivation for this change
New package wg-netmanager, which maintains a net connected via wireguard

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/wg-netmanager/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Test compilation/test with:
+#   nix build -f ~/src/nixpkgs wg-netmanager
+#
+# Test execution with:
+#   nix-env -f ~/src/nixpkgs -iA wg-netmanager
```
Comment (legendofmiracles) [inline pkgs/tools/networking/wg-netmanager/default.nix:5]: This is not needed...

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -4361,6 +4361,12 @@
     githubId = 27668;
     name = "Tobias Pflug";
   };
+  gin66 = {
+    email = "jochen@kiemes.de";
+    github = "gin66";
+    githubId = 5549373;
+    name = "Jochen Kiemes";
+  };
```
Comment (legendofmiracles) [inline maintainers/maintainer-list.nix:4369]: Different commit: `maintainers: add gin66`

File: pkgs/tools/networking/wg-netmanager/default.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    sha256 = "16g5dfd4w0qx7pcgx6xw0py6w0s3dxwfb53j1xpsiz81412b71i4";
+  };
+
+  # related to the Cargo.lock file
+  cargoSha256 = "0f0qd6l3ra64rny2zsmsj4x3zrilqhnlp4l2l5sqglgkxg8g17jz";
+
+  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
+
+  # Testing this project requires sudo, Docker and network access, etc.
+  doCheck = true;
```
Comment (legendofmiracles) [inline pkgs/tools/networking/wg-netmanager/default.nix:26]: The comment and code seem contradictory at first, maybe move the comment down to the checkFlags attribute?

**Comment by gin66** [comment]
I will create a new pull request with the two commits as requested and an updated version of the package.

**Comment by legendofmiracles** [comment]
FYI: you can modify everything locally and then override what you pushed before with the --force flag in git push. 

No need to create a new PR, but I do appreciate the enthusiasm

**Comment by gin66** [comment]
actually I am not aware of the —force feature.
I only know, how to delete a tag.
Thanks for your hint and guidance

**Pull request closed without merging**